src/foam/u2/wizard/views/FocusWizardView.js

```diff
@@ -88,7 +88,7 @@ foam.CLASS({
           return data$currentWizardlet.subTitle ?
             this.E().start()
               .addClasses([self.myClass('wizardletTitle'), 'p', self.myClass('wizardletSub')])
-              .add(data$currentWizardlet.subTitle)
+              .tag(foam.u2.HTMLView.create({ nodeName: 'div', data: data$currentWizardlet.subTitle }))
```

Comment:
This is going to break some css cause HTMLView adds it's own padding, that should be removed here